hw/ip/sram_ctrl/doc/dv/index.md

```diff
@@ -100,11 +100,30 @@ The SRAM_CTRL RAL model is created with the [`ralgen`]({{< relref "hw/dv/tools/r
 
 It can be created manually by invoking [`regtool`]({{< relref "util/reggen/README.md" >}}):
 
-<!-- TODO add description of second RAL model once implemented -->
+Besides default RAL that provides the access to CSRs, there is another RAL for SRAM memory.
+This RAL is in `sram_ctrl_prim_ral_pkg.sv` which is added manually and enabled with all the common CSR/memory tests, such mem_walk, tl_errors, tl_intg_err etc.
 
 ### Reference models
-The SRAM_CTRL testbench uses a SystemVerilog golden model of its address and data scrambling to ensure that internal encryption and decryption are performed correctly.
+There are 3 DV reference models used in this testbench.
+- The SRAM_CTRL testbench uses a SystemVerilog golden model of its address and data scrambling to ensure that internal encryption and decryption are performed correctly.
```

Comment:
```suggestion
- The SRAM_CTRL testbench uses a SystemVerilog golden model for address and data scrambling to ensure that internal encryption and decryption perform correctly.
```
Do you think this is better? Just a suggestion :)

Reply:
Updated, thanks